praw/models/user.py

```diff
@@ -172,3 +172,18 @@ def subreddits(
         return ListingGenerator(
             self._reddit, API_PATH["my_subreddits"], **generator_kwargs
         )
+
+    def trusted(self) -> List["praw.models.Redditor"]:
+        """Return a RedditorList of trusted Redditors.
+
+        To display the names of your trusted users and the times at which you decided to trust
+            them, try:
```

Comment:
```suggestion
        To display the usernames of your trusted users and the times at which you decided to
            trust them, try:
```
Can we use usernames instead of names? Also, does the second line need indented?